### PR TITLE
raidboss: e12s chiseled sculpture triggers (alternative)

### PR DESCRIPTION
**Pull request by quisquous**
We were running a bunch of e12s and so I had time to test this out
and iron it out.

I personally found that having a trigger on:
* initial number
* left right
* laser bait
* other three lasers
* knockback location

...was just incredibly noisy.

I have cut this back from #2443 to say:
* initial number (inner outer)
* left/right OR left/right + bait #1
* bait #n laser (no other lasers called out)
* knockback location

Inner/Outer seems to work with the changes that I made here to collect
the sculptures separately from the tether.

(@valarnin, sorry to implement a patch that you were in the middle
of, but I wanted to get this in before I didn't have a chance to
test it anymore.)

Closes #2443.

### CODE REVIEW

**Comment by quisquous** [comment]
https://clips.twitch.tv/HonestDistinctGooseMingLee-NZRMW8o0n-jzQkHn

**Comment by valarnin** [comment]
Ultimately my goal was to get the triggers into cactbot in general, I'm glad that my original pass at it was sufficient to get there, even if in a modified form.

I tend to err on the side of more callouts with the understanding that it's easier to pare them down than to add additional later. I agree that it was potentially a bit excessive in this case.

In the end this is one less trigger I'll have to maintain for my static in the future, and one more trigger that everyone can benefit from.

File: ui/raidboss/data/05-shb/raid/e12s.js [hunk 1]
```diff
@@ -400,43 +400,189 @@ export default {
...
+    {
+      id: 'E12S Promise Chiseled Sculpture Collector',
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '9818' }),
+      run: (data, matches) => {
+        // Collect both sculptures up front, so when we find the tether on the
+        // current player we can look up both of them immediately.
+        data.statueIds = data.statueIds || [];
+        data.statueIds.push(parseInt(matches.id, 16));
+      },
+    },
```
Comment (valarnin) [inline ui/raidboss/data/05-shb/raid/e12s.js:425]: This is the only thing that jumps out at me as iffy. I'm not sure if the log lines for this happen at fade in or during the fight, if it's the latter then the trigger will fail if it's started after zone-in but before pull.

Comment (quisquous) [reply ui/raidboss/data/05-shb/raid/e12s.js:425]: These happen mid-fight:
```
[20:07:59.272] 03:4000900B:Added new combatant Chiseled Sculpture.  Job: N/A Level: 80 Max HP: 148000 Max MP: 10000 Pos: (0,-75,75) (98180000012445).
[20:07:59.272] 03:4000900A:Added new combatant Chiseled Sculpture.  Job: N/A Level: 80 Max HP: 148000 Max MP: 10000 Pos: (0,-75,75) (98180000012445).
[20:08:04.397] 16:4000900B:Chiseled Sculpture:58B2:Classical Sculpture:etc
```

Comment (valarnin) [reply ui/raidboss/data/05-shb/raid/e12s.js:425]: Alright. I wasn't at my computer to verify this at the time.

Comment (quisquous) [reply ui/raidboss/data/05-shb/raid/e12s.js:425]: Thanks for the question.  There's been a lot of things added early and then moved later this tier, so it's good to check.  :)

**Pull request merged**